packages/wrangler/src/index.ts

```diff
@@ -229,7 +229,7 @@ export function createCLIParser(argv: string[]) {
 		)
 		.option("env", {
 			alias: "e",
-			describe: "Environment to use for operations and .env files",
+			describe: "Environment to use for operations, .env and .dev.vars files",
```

Comment:
Strictly the `.dev.vars` files are only used for

- `wrangler dev`
- `wrangler types`
- `getPlatformProxy()`

Not sure if this is important to call out here or somewhere?